src/components/atoms/Hamburguer/Hamburguer.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, cleanup } from '@testing-library/react';
+import Hamburguer from './index';
+
+describe('Test Hamburguer Component', () => {
+  afterEach(cleanup);
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<Hamburguer src="/menu.svg" alt="Menu Icon" />, div);
+  });
+
+  it('render correct image', () => {
+    const { getByAltText } = render(<Hamburguer />);
+    expect(getByAltText('Menu Icon')).toHaveAttribute('src', '/menu.svg');
```

Comment:
`getByRole('image', {name: /alt text here/i})` query has higher priority in RTL.

Reply:
Now using getByRole as suggested :)